SCClassLibrary/Common/GUI/tools/Stethoscope.sc

```diff
@@ -393,7 +401,20 @@ Stethoscope {
 
 	size_ { arg value;
 		var sz = value.asSize;
-		if( window.notNil ) { window.setInnerExtent(sz.width,sz.height) };
+		if(window.notNil) {
```

Comment:
If this change is approved, this behavior should be documented: it changes the window position in favor of keeping the window fully on-screen with specified size.

Reply:
Thanks, done:
https://github.com/supercollider/supercollider/pull/6230#issuecomment-1987043337